psscriptanalyzer/Invoke-HealthCheck.ps1

```
@@ -0,0 +1,62 @@
+      } else {
+          Write-Host "$url -> $($resp.StatusCode)"
+      }
+  }
+}
+
+function Process-DiagnosticBundle {
+    param(
+        [string]$Username,
+        [string]$Password
+    )
+    $secure = ConvertTo-SecureString -String $Password -AsPlainText -Force
```

PSScriptAnalyzer (1.25.0)

[error] File 'Invoke-HealthCheck.ps1' uses ConvertTo-SecureString with plaintext. This will expose secure information. Encrypted standard strings should be used instead.

(PSAvoidUsingConvertToSecureStringWithPlainText)